poly-commitment/benches/ipa.rs

```diff
@@ -71,6 +71,52 @@ fn benchmark_msm(c: &mut Criterion) {
     }
 }
 
+fn benchmark_msm_parallel(c: &mut Criterion) {
```

Comment:
Nit: I would add Vesta in the name.

Reply:
Nit: same than https://github.com/o1-labs/proof-systems/pull/2969#discussion_r1965248308

Reply:
Nit: same than https://github.com/o1-labs/proof-systems/pull/2969#discussion_r1965248308